src/stores/user.store.ts

```diff
@@ -13,12 +13,21 @@ export const useUserStore = defineStore('user', () => {
 	const ToastStore = useToastStore()
 	const ErrorStore = useErrorStore()
 	const inscriptions = ref({})
+	const MailVerified = ref (false)
 	const { setContent }  = ToastStore;
 	const { add_error } = ErrorStore;
 	async function get_csrf() {
 		const response = await axios.get('/user/get-csrf/')
 	}
 
+
+	//C'ets une dingz mais promis ça va marcher :
+	async function verifMail(idname: String , idtoken: String){
+		MailVerified.value=false
+		let result = await axios.get(`/user/confirm/${idname}/${idtoken}`)
```

Comment:
This line is working but the best practice in js is to use `const` instead of `let` if you don't reassign it's value later.

This can raise a linter error or warning (we don't have one currently, Dyskal is working on it).

Reply:
i don't even use the value so i'll just delete result